### ISSUE TITLE
Working Session Proposal: Value of identifiers in datasets 

### ISSUE THREAD

**Issue by irummaqsood** [OpenDataServices/iodc-data-standards-day-2018#1]
# Value of identifiers in datasets 
Session objective: This workshop invites data consumers and publishers to explore the benefits/challenges of adding unique identifiers in datasets. In addition participants will discuss various strategies to increase the implementation of identifiers in datasets and to ensure they are implemented consistently by using open guidelines/standards.

Identifiers are crucial to how data can be effectively published, retrieved, reused and linked. Identifiers implemented correctly allow data consumers to form connections between datasets whereas incorrect/lack of implementation of identifiers stifles the power of information gained from these identifiers. 

Improper usage or lack of implementation of identifiers in datasets also cause interoperability issues which hinder users from translating data into meaningful insights. 
Implementing Identifiers in datasets are important however implementing these identifiers consistently in large organizations such as Government departments comes with unique challenges associate to each user group (data publishers, data users). Data publishers have challenges implementing identifiers in datasets, these could be as easy as, not knowing which Organization identifier (department number) to use and as complex as how to name a identifier properly. 
There is a lack of open standards and guidelines that provide data publishers with a) publishing an authoritative list of consistent identifiers (example Organizations) so other datasets can reference them  b) how to apply the identifiers consistently across datasets. c) standards on how identifiers should be developed (example creating an identifier for departments that constantly change names).

In this workshop data consumers and data publishers will discuss challenges they have faced implementing the identifiers in datasets and how they have overcome these challenges. Data publishers will discuss various strategies to increase the adoption of identifiers in datasets and guidelines/standards that they have used for implementing identifiers in datasets. 

The overall goal of the session is to highlight some of the challenges/benefits associated with implementation of the identifiers in datasets, and to increase the appetite of data publishers on adoption of identifiers in datasets. 

## Format Details
In a brief introduction, workshop moderator will introduce the session concept, discussing an actual use case of implementing identifiers and showcasing different methods to address the challenges associated with implementing identifiers in datasets (15 min).

Moderator will invite the group to identify challenges they have faced implementing identifiers in datasets and will document the challenges participants have identified. 

Moderator will group the challenges into themes and will split the group into themes. In the breakout sessions participants will be asked to brainstorm on how to address the challenges associated to implementing identifiers and suggest possible solutions.  

In a concluding round (10 min) participants will share their experiences, and how well each approach suggested meets their requirements to implement identifiers in dataset.

After the session moderator will document the findings and will publish a blog post on possible challenges and their solutions for implementing unique identifiers in datasets.

**Comment by timgdavies** [comment]
This sounds like a great session idea. Would love to contribute learning from the org-id.guide project as well.

**Comment by lgrino** [comment]
Agreed! Very good to see a session on this critical issue. Foundation Center is part of a project called BRIDGE, which assigns unique identifiers to social sector organizations around the world. We'd love to share information about the project and our experiences as well.
